client/src/pages/Edit.js

```diff
@@ -66,98 +68,90 @@ function Edit() {
   };
 
   return (
-    <div>
+    <>
       <Container>
         <h1>Updated Food Items!</h1>
         <div>
           <form onSubmit={FormHandle}>
             <div>
-              <label>Food Name</label>
+              <label>Food_Title</label>
```

Comment:
Do you think it would look better if you used a space in the label instead of an underscore?

```suggestion
              <label>Food Title</label>
```